learn/getting-started/deploying-ballerina-on-kubernetes.md

```diff
@@ -12,15 +12,18 @@ redirect_from:
 
 In this section, you will be writing a simple Ballerina service and then, you will Dockerize the application and deploy it in Kubernetes.
 
-## Prerequisites
+## Setting up the Prerequisites
```

Comment:
This is redundant IMO

Reply:
Same as this [comment](https://github.com/ballerina-platform/ballerina-dev-website/pull/3564/files#r794989678).